presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -9,287 +9,458 @@ Except for the limit on queued queries, when a resource group runs out of a reso
+
+This allows you to prioritize certain subgroups over others in terms of resource allocation.
+In this example, queries from the "engineering" subgroup will be prioritized over queries
+from the "marketing" subgroup.
+
+... literalinclude:: schedule-weight-example.json
+:   :language: text
 
 Selector Rules
 --------------
 
-* ``user`` (optional): regex to match against user name.
+Here are the key components of selector rules in PrestoDB:
 
-* ``source`` (optional): regex to match against source string.
+* ``user`` (optional): This is a regular expression that matches the user who is submitting the query.
+
+* ``userGroup`` (optional): regex to match against every user group the user belongs to.
+
+* ``source`` (optional): This matches the source of the query, which is typically the
+  application submitting the query.
+
+* ``queryType`` (optional): This specifies the type of the query such as READ, WRITE, or MANAGE.
+  String to match against the type of the query submitted:
 
-* ``queryType`` (optional): string to match against the type of the query submitted:
-    * ``DATA_DEFINITION``: Queries that alter/create/drop the metadata of schemas/tables/views, and that manage
-      prepared statements, privileges, sessions, and transactions.
-    * ``DELETE``: ``DELETE`` queries.
-    * ``DESCRIBE``: ``DESCRIBE``, ``DESCRIBE INPUT``, ``DESCRIBE OUTPUT``, and ``SHOW`` queries.
-    * ``EXPLAIN``: ``EXPLAIN`` queries.
-    * ``INSERT``: ``INSERT`` and ``CREATE TABLE AS`` queries.
     * ``SELECT``: ``SELECT`` queries.
+    * ``EXPLAIN``: ``EXPLAIN`` queries (but not ``EXPLAIN ANALYZE``).
+    * ``DESCRIBE``: ``DESCRIBE``, ``DESCRIBE INPUT``, ``DESCRIBE OUTPUT``, and ``SHOW`` queries.
+    * ``INSERT``: ``INSERT``, ``CREATE TABLE AS``, and ``REFRESH MATERIALIZED VIEW`` queries.
+    * ``UPDATE``: ``UPDATE`` queries.
+    * ``DELETE``: ``DELETE`` queries.
+    * ``ANALYZE``: ``ANALYZE`` queries.
+    * ``DATA_DEFINITION``: Queries that alter/create/drop the metadata of schemas/tables/views,
+      and that manage prepared statements, privileges, sessions, and transactions.
 
 * ``clientTags`` (optional): list of tags. To match, every tag in this list must be in the list of
   client-provided tags associated with the query.
 
-* ``group`` (required): the group these queries will run in.
+* ``group`` (required): This is the name of the resource group where the query will be routed.
+
+Selectors are processed sequentially and the first one that matches will be used.
 
 Global Properties
 -----------------
 
 * ``cpuQuotaPeriod`` (optional): the period in which cpu quotas are enforced.
+  The cpuQuotaPeriod is a global property often used in container-based environments to control the amount
+  of CPU resources that a container can use in a specified period.
 
-Selectors are processed sequentially and the first one that matches will be used.
+This property is typically used in conjunction with a cpuQuota property,
```

Comment:
Is ``cpuQuota`` also a global property like ``cpuQuotaPeriod``? If yes, consider revising this line to add ``cpuQuota`` as a new unordered list item. 

Also, consider adding a line explaining what the values are for these two properties. Maybe something like "The value of ``cpuQuota`` and ``cpuQuotaPeriod`` is the number of microseconds of CPU time."